frontend/app/src/people/utils/PaidBounty.tsx

```diff
@@ -50,7 +50,7 @@ const PaidBounty = (props: PaidBountiesProps) => {
           title={props.title}
           codingLanguage={props.codingLanguage}
           isPaid={true}
-          org_img={props.img}
+          org_img={props.org_img} //added to fix 1082
```

Comment:
instead of adding a comment referencing a github issue since we cant control if github deletes issues incase we need to move to a different platform like gitlab could you change this to something like

`org_img={props.org_img} //This sets the organization image in bounty`

if you could do that for the other comments aswell